webhook-pubsub-publisher/README.md

```diff
@@ -0,0 +1,52 @@
+# Webhook Pub/Sub Publisher
```

Comment:
I think @rileyajones would appreciate it if you added deployment steps

Reply:
The deployment will use the same method in DEPLOYMENT.md, so I'm just adding a short description like in the readme for pr-deploy or bundle-size. Done.